### ISSUE TITLE
OLAP Extensions not supported by AtScale

### ISSUE THREAD

**Issue by PriteshGehlot** [OlapPivotTableExtensions/OlapPivotTableExtensions#25]
I get OLAP Extension error while trying to use this extension in AtScale cube. Can you please see if this can be fixed.. See huge potential

**Comment by furmangg** [comment]
Currently only Analysis Services and PowerPivot are supported. I will leave this open as a feature request in case we could add support in the future.

**Comment by garrybd6** [comment]
It would be great to have this supported on the AtScale XMLA Endpoint. I'm happy to help make contact at AtScale if needed to add in support.

**Comment by kburoker0328** [comment]
Agreed. Business users heavily use the "Filter" functionality of their OLAP Extension and we are transitioning to AtScale. Can we get this supported by AtScale?

**Comment by furmangg** [comment]
Thanks for reaching out. AtScale support is something I will consider but no immediate plans. 

Would love to discuss further if all of you would like to reach out to me privately:

https://olappivottableextensions.github.io/Contact
